File: FINALpython/copies/8.c
#include <stdio.h>
#include <string.h>
void mul(int a, int b) {

        printf("%d\n", a * b);

}
void convert2(char *st1, char *st2) {
	char *ones[19] = {"one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten", "eleven", "twelve", "thirteen", "fourteen", "fifteen", "sixteen", "seventeen", "eighteen", "nineteen"};
	char *tens[8] = {"twenty", "thirty", "fourty", "fifty", "sixty", "seventy", "eighty", "ninety"};
	int numb1 = 0;
	if (strstr(st1, "-") != NULL) {
		char numberst[30];
		strcpy(numberst, st1);
		char *part;
		part = strtok(numberst, "-");
		int a;
		for (a = 0; a < (int)sizeof(tens)/sizeof(tens[0]); a++) {
			if (strcasecmp(part, tens[a]) == 0) {
				numb1 += 10 * (a + 2);
				break;
			}
		}	
		part = strtok(NULL, "-");
		int b;
		for (b = 0; b < (int)sizeof(ones)/sizeof(ones[0]); b++) {
			if (strcasecmp(part, ones[b]) == 0) {
				numb1 += b + 1;
				break;
			}
		}
	} else {
		int i;
		int check = 0;
		for (i = 0; i < (int)sizeof(ones)/sizeof(ones[0]); i++){
			if(strcasecmp(st1, ones[i]) == 0) {
				numb1 += (i + 1);
				check = 1;
				break;
			}
		}
		if (!check) {
			int j;
			for(j = 0; j < (int)sizeof(tens)/sizeof(tens[0]); j++) {
				if (strcasecmp(st1, tens[j]) == 0) {
					numb1 += 10 * (j + 2);
					break;
				}
			}
		}
	}
	int numb2 = 0;
        if (strstr(st2, "-") != NULL) {
                char numberst[30];
                strcpy(numberst, st2);
                char *part;
                part = strtok(numberst, "-");
                int a;
                for (a = 0; a < (int)sizeof(tens)/sizeof(tens[0]); a++) {
                        if (strcasecmp(part, tens[a]) == 0) {
                                numb2 += 10 * (a + 2);
                                break;
                        }
                }
                part = strtok(NULL, "-");
                int b;
                for (b = 0; b < (int)sizeof(ones)/sizeof(ones[0]); b++) {
                        if (strcasecmp(part, ones[b]) == 0) {
                                numb2 += b + 1;
                                break;
                        }
                }
        } else {
                int i;
                int check = 0;
                for (i = 0; i < (int)sizeof(ones)/sizeof(ones[0]); i++){
                        if(strcasecmp(st2, ones[i]) == 0) {
                                numb2 += (i + 1);
                                check = 1;
                                break;
                        }
                }
                if (!check) {
                        int j;
                        for(j = 0; j < (int)sizeof(tens)/sizeof(tens[0]); j++) {
                                if (strcasecmp(st2, tens[j]) == 0) {
                                        numb2 += 10 * (j + 2);
                                        break;
                                }
                        }
                }
        }
	mul(numb1,numb2);
}
void convert(char *st) {
	char n[30];
	strcpy(n, st);
        char *st1 = strtok(n, " ");
	char *st2 = strtok(NULL, " ");
        convert2(st1, st2); 

}

int main() {
	convert("Five three");
	convert("Twenty fifteen");		
}

File: FINALc/copies/8.py
#!/usr/bin/python

ones = ["one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten", "eleven", "twelve", "thirteen", "fourteen", "fifteen", "sixteen", "seventeen", "eighteen", "nineteen"]
tens = ["twenty", "thirty", "fourty", "fifty", "sixty", "seventy", "eighty", "ninety"]

word = raw_input("Enter two word to multiply: ")
ws = word.split(' ')
fw = ws[0].split('-')
sw = ws[1].split('-')

if len(fw)< 2:
	for x in range(len(ones)):
		if fw[0] == ones[x] or fw[0].lower() == ones[x]:
			a = x + 1
	for y in range(len(tens)):
                if fw[0] == tens[y] or fw[0].lower() == tens[y]:
                        a = (y + 2) * 10
else:
	for x in range(len(tens)):
                if fw[0] == tens[x] or fw[0].lower() == tens[x]:
                        t = (x + 2) * 10
        for y in range(len(ones)):
                if fw[1] == ones[y] or fw[1].lower() == ones[y]:
			o = y + 1
	a = t + o

if len(sw)< 2:
        for x in range(len(ones)):
                if sw[0] == ones[x] or sw[0].lower() == ones[x]:
                        b = x + 1
        for y in range(len(tens)):
                if sw[0] == tens[y] or sw[0].lower() == tens[y]:
                        b = (y + 2) * 10
else:
        for x in range(len(tens)):
                if sw[0] == tens[x] or sw[0].lower() == tens[x]:
                        t = (x + 2) * 10
        for y in range(len(ones)):
                if sw[1] == ones[y] or sw[1].lower() == ones[y]:
                        o = y + 1
        b = t + o

print(a * b)


